.travis.yml

```diff
@@ -5,6 +5,8 @@ env:
     - CC_TEST_REPORTER_ID=d570e2625aaffa20e7828eaddfbec748c525e866b6d413472bc26e188cb2bc6a
     - DATABASE_URL_TEST=postgres://postgres@localhost:5432/authorhavens_test
     - APP_URL=http://localhost:3000
```

Comment:
why have both `APP_URL` and `APP_URL_BACKEND`

Reply:
@e-liyai  As I didn't want to break others' code until they change the name of this env variable, I just let it there till all of us use it then I'll remove it. The reason why we opted for this naming is that when having emails sent from the backend, we want most of the links in those emails to redirect to the frontend app.